bridges/relays/utils/src/initialize.rs

```diff
@@ -18,7 +18,12 @@
 
 use console::style;
 use parking_lot::Mutex;
-use std::{cell::RefCell, fmt::Display, io::Write};
+use std::{cell::RefCell, fmt::Display};
```

Comment:
I don't like what's happening here :-)
Please revert these changes.

Let's keep only the changes that can be done using the one-liners init_for_tests and try_init_simple in this PR. All other changes should be done in a separate PR.

Reply:
This is fine IMO, it's clear/simple/easy change aligned with the plan to replace `env_logger` with `sp_tracing`.

@serban300 can make final decision as the code-owner of this file.

Reply:
Looks good to me as well

Reply:
Ok then :-)